From c27d9dc43f1ceefe4d64fc0a135b9e95ed944c07 Mon Sep 17 00:00:00 2001
From: Woomymy <woomy@woomy.be>
Date: Sun, 22 Jan 2023 13:09:12 +0100
Subject: [PATCH] lineage-sdk/settings: Don't turn on restricted networking
 mode by default

"Fixes" some networking issues caused by our prebuilt MTK kernel
with broken BPF

For more information: https://forum.xda-developers.com/t/gsi-13-lineageos-20-light.4509315/post-87620773

Change-Id: Ic0a3ebb64cccaa5657c1c2c0a2f50daa8ff0f067
Signed-off-by: Woomymy <woomy@woomy.be>
---
 .../org/lineageos/lineagesettings/LineageDatabaseHelper.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java b/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
index 52d97a9..3ea6a44 100644
--- a/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
+++ b/packages/LineageSettingsProvider/src/org/lineageos/lineagesettings/LineageDatabaseHelper.java
@@ -581,7 +581,7 @@ public class LineageDatabaseHelper extends SQLiteOpenHelper{
 
     private void loadRestrictedNetworkingModeSetting() {
         Settings.Global.putInt(mContext.getContentResolver(),
-                Settings.Global.RESTRICTED_NETWORKING_MODE, 1);
+                Settings.Global.RESTRICTED_NETWORKING_MODE, 0);
         try {
             List<PackageInfo> packages = new ArrayList<>();
             for (UserInfo userInfo : UserManager.get(mContext).getAliveUsers()) {
-- 
2.39.1

